app/services/CalculationService.scala

```diff
@@ -215,18 +215,18 @@ trait CalculationService {
     allowableLossesAmt: Double,
     annualExemptAmount: Double,
     broughtForwardLosses: Double = 0
-  ): Double = (
+  ): Double =
     gain match {
       case a if a <= 0 => a //gain less than 0, no need to deduct reliefs, losses or aea
       case b => b - round("up", reliefs) match { //gain greater than 0 so deduct the reliefs
         case c if c <= 0 => 0 //Reliefs cannot turn gain into a loss, hence return 0
         case d => d - round("up", allowableLossesAmt) match { //Gain greater than 0 still so deduct allowable loses
-          case e if e <= 0 => e //Allowable losses turns gain into a loss so return the loss
-          case f => negativeToZero(f - round("up", annualExemptAmount)) //deduct AEA, if amount less than 0 return 0 else return amount
+          case e if e <= 0 => e - round("up", broughtForwardLosses) //Allowable losses turns gain into a loss so return the loss
+          case f => negativeToZero(f - round("up", annualExemptAmount)) - round("up", broughtForwardLosses) //deduct AEA, if amount less than 0 return 0 else return amount
         }
       }
     }
-  ) - round("up", broughtForwardLosses) // Finally, subtract any brought forward losses.
+   // Finally, subtract any brought forward losses.
```

Comment:
Good spot moving this, but could you move this comment and add to line 25 to reflect what it is doing